dev-docs/roadmaps/api-evolution-roadmap.md

```diff
@@ -0,0 +1,55 @@
+# API Evolution Roadmap
+
+The deck.gl core API will need to be modernized over time. Changes to the core API have a big impact, this roadmap ensures that we have a clear long-term goal to work towards.
+
+
+## Core Principles
+
+### ONE API Principle
+
+deck.gl is designed based on a "ONE API" principle, which is intended as a strong encouragement to API designers to ensure that we do not create irreconcilable forks in our API, as we continue to improve support for the React, Pure JS, scripting, JSON etc versions of the deck.gl API. Ideally a program should be portable between versions with very small differences, and learning how to use deck.gl in one version should translate directly to all versions.
+
+Therefore any proposals here need to consider this principle and justify the design in case of deviations. Of course, by taking the conversation one step up, and talking about imperative vs. reactive/functional paradigms we can argue that this support does not cater to one specific API version but rather to ensure that we support multiple programming paradigms. Still, these changes will make programs written in the various deck.gl API incarnations more different, so it is good to exercise constraint when possible.
+
```

Comment:
Duplicated with the `## API Design Constraints section` in imperative-api-rfc.md. 
Was it on purpose?

Reply:
> Duplicated with the ## API Design Constraints section in imperative-api-rfc.md. Was it on purpose?

Deduplicated